src/lifetimes.md

```diff
@@ -213,3 +213,204 @@ totally ok*, because it keeps us from spending all day explaining our program
 to the compiler. However it does mean that several programs that are totally
 correct with respect to Rust's *true* semantics are rejected because lifetimes
 are too dumb.
+
+# Inter-procedural Borrow Checking of Function Arguments
+
+Consider the following program:
+
+```rust,ignore
```

Comment:
Is there any reason why this is `ignore`? Seems like it should be compilable and runable, no?